sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxSqaleXmlLoader.java

```diff
@@ -0,0 +1,186 @@
+/*

```

Comment:
sucks a bit, in this case we need to have a integration test for this. or we might have the same issue as it happen with 0.9.4